modules/android/pages/p2psync-multipeer.adoc

```diff
@@ -1,10 +1,291 @@
-= Multipeer Replicator
+= Multipeer P2P Replicator
 ifdef::show_edition[:page-edition: Enterprise Edition]
-:description: MultipeerReplicator is a peer-to-peer replicator that can advertise its presence, discover other peers, establish connections, and replicate data with those peers.
-
+:description: The Multipeer P2P Replicator enables lightweight, self-organizing mesh networks for apps running on the same local Wi-Fi.
 :source-language: Java
```

Comment:
Working on the iOS version of this now, with a code sample from Pasin - will upload a draft tomorrow.